curriculum/challenges/english/01-responsive-web-design/css-grid/place-items-in-grid-areas-using-the-grid-area-property.english.md

```diff
@@ -30,7 +30,7 @@ Place an element with the <code>item5</code> class in the <code>footer</code> ar
 ```yml
 tests:
   - text: <code>item5</code> class should have a <code>grid-area</code> property that has the value of <code>footer</code>.
-    testString: assert(code.match(/.item5\s*?{[\s\S]*grid-area\s*?:\s*?footer\s*?;[\s\S]*}/gi));
+    testString: const removeCssComments = str => str.replace(/\/\*[\s\S]+?\*\//g, '');assert(removeCssComments(code).match(/.item5\s*?{[\s\S]*grid-area\s*?:\s*?footer\s*?;[\s\S]*}/gi));
```

Comment:
This challenge passes on production without removing the comments for me. I don't think it's necessary to add this here, but I wouldn't object to leaving it. I think you mistook this file with the one in the issue which is [divide the grid into an area template](https://www.freecodecamp.org/learn/responsive-web-design/css-grid/divide-the-grid-into-an-area-template)

Reply:
@moT01 I see what you mean about this particular challenge.  I only added it because the issue said it failed with the comment left in.  I did not test this one without removing the comment, but I did on the other challenge.

Reply:
I think you are misunderstanding. The issue does not say that this challenge is failing with the comment left in. The challenge in the issue is the one I linked above, which is not included in this PR.

Reply:
@moT01 Thanks for catching that!  I have added a new commit to add the function call to the `Divide the Grid Into an Area Template` challenge.